packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-pwd-recovery-mfa/java/summaryofsteps.md

```diff
@@ -0,0 +1,9 @@
+## Summary of steps
+
+The following are the sequence of steps for the password recovery use case.
```

Comment:
"following are the sequence" seems a bit awkward to me. How about "following diagram shows the sequence" ?

Reply:
Perfect; can be resolved.